classes/webservice/WebserviceRequest.php

```diff
@@ -183,7 +183,7 @@ class WebserviceRequestCore
     public static $ws_current_classname;
 
 
-    public static $shopIDs;
+    public static $shopIDs = array();
```

Comment:
would you mind to add PHP docs and set the value to an empty array so we'll never do the same mistake again?

```php
    /**
     * @var array A list of Shop ids, can be empty.
     */
    public static $shopIDs = [];
```